install/resources/mybb_theme.xml

```diff
@@ -3639,36 +3639,15 @@ var announcement_quickdelete_confirm = "{$lang->announcement_quickdelete_confirm
 </div>
 <br />]]></template>
 		<template name="error_inline_item" version="1818"><![CDATA[<li>{$error}</li>]]></template>
-		<template name="error_nopermission" version="1818"><![CDATA[{$lang->error_nopermission_guest_1}
+		<template name="error_nopermission" version="1824"><![CDATA[{$lang->error_nopermission_guest_1}
 <ol>
 <li>{$lang->error_nopermission_guest_2}</li>
```

Comment:
```suggestion
<li>{$lang->error_nopermission_guest_2} <a href="member.php?action=login">{$lang->login}</a> | <a href="member.php?action=register">{$lang->need_reg}</a></li>
```

I think it's best to add some links here and change the language line:
https://github.com/mybb/mybb/blob/36e8e7167c693ad0ac78dd05406e4cfa8fe0a9eb/inc/languages/english/messages.lang.php#L54